crates/libs/core/src/imp/factory_cache.rs

```diff
@@ -16,6 +16,12 @@ impl<C, I> FactoryCache<C, I> {
     }
 }
 
+impl<C, I> Default for FactoryCache<C, I> {
+    fn default() -> Self {
+        Self::new()
+    }
+}
+
```

Comment:
Is this needed in this PR?

Reply:
https://github.com/microsoft/windows-rs/pull/2898/commits/51842acb888559f2923d398e3e93f137a5017536 deals with a new nightly Clippy warning.